polyjuice-tests/src/test_cases/receive_ether.rs

```diff
@@ -0,0 +1,86 @@
+use ckb_vm::Bytes;
+use gw_common::state::State;
+use gw_generator::traits::StateExt;
+use gw_store::{chain_view::ChainView, traits::chain_store::ChainStore};
+use gw_types::{
+    packed::RawL2Transaction,
+    prelude::{Builder, Entity, Pack},
+};
+
+use crate::helper::{
+    check_cycles, create_block_producer, deploy, new_block_info, parse_log, setup,
+    MockContractInfo, PolyjuiceArgsBuilder, CREATOR_ACCOUNT_ID, L2TX_MAX_CYCLES,
+};
+
+const INIT_CODE: &str = include_str!("./evm-contracts/EtherReceiverMock.bin");
+
+#[test]
+fn receive_ether_test() -> anyhow::Result<()> {
```

Comment:
Please add a test case:
If neither a receive Ether nor a payable fallback function is present, the contract cannot receive Ether through regular transactions and throws an exception.

Reply:
done